src/app/clusters/content-launch-server/content-launch-server.cpp

```diff
@@ -53,6 +62,60 @@ ContentLaunchResponse contentLauncherClusterLaunchContent(chip::EndpointId endpo
 ContentLaunchResponse contentLauncherClusterLaunchUrl(const chip::CharSpan & contentUrl, const chip::CharSpan & displayString,
                                                       ContentLaunchBrandingInformation & brandingInformation);
 
+namespace {
+
+class ContentLauncherAttrAccess : public app::AttributeAccessInterface
+{
+public:
+    ContentLauncherAttrAccess() : app::AttributeAccessInterface(Optional<EndpointId>::Missing(), app::Clusters::ContentLauncher::Id)
+    {}
+
+    CHIP_ERROR Read(const app::ConcreteReadAttributePath & aPath, app::AttributeValueEncoder & aEncoder) override;
+
+private:
+    CHIP_ERROR ReadAcceptsHeaderAttribute(app::AttributeValueEncoder & aEncoder);
+    CHIP_ERROR ReadSupportedStreamingProtocols(app::AttributeValueEncoder & aEncoder);
+};
+
+ContentLauncherAttrAccess gContentLauncherAttrAccess;
+
+CHIP_ERROR ContentLauncherAttrAccess::Read(const app::ConcreteReadAttributePath & aPath, app::AttributeValueEncoder & aEncoder)
+{
+    switch (aPath.mAttributeId)
+    {
+    case app::Clusters::ContentLauncher::Attributes::AcceptsHeaderList::Id: {
+        return ReadAcceptsHeaderAttribute(aEncoder);
+    }
+    case app::Clusters::ContentLauncher::Attributes::SupportedStreamingProtocols::Id: {
+        return ReadSupportedStreamingProtocols(aEncoder);
+    }
+    default: {
+        break;
+    }
+    }
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ContentLauncherAttrAccess::ReadAcceptsHeaderAttribute(app::AttributeValueEncoder & aEncoder)
```

Comment:
I do not think it is a good idea to hardcode this value inside SDK, it is not an example here.

Please have a look at examples/tv-app/android/java/ContentLauncherManager.cpp, we have already added related code to the example so that it can give it`s own values.

Reply:
The related code is here
https://github.com/project-chip/connectedhomeip/blob/master/examples/tv-app/android/java/ContentLauncherManager.cpp

Reply:
Updated.